android/app/src/main/res/values/strings.xml

```diff
@@ -30,6 +30,8 @@
     <!-- ??? -->
     <string name="one">1</string>
     <string name="delete">Delete</string>
+    <string name="dump">Dump</string>
+    <string name="dumpAll">Dump All</string>
```

Comment:
Again... I'd like to keep style consistent.  There is an ancient reason, now probably irrelevant, for using lower-with-underscore names.  Unless there is an actual reason for doing otherwise, lets stick with it.